src/pages/goodsDetail/GoodsDetail.js

```diff
@@ -0,0 +1,43 @@
+import React, { useEffect, useState } from 'react';
+import styled from 'styled-components';
+import LeftContents from './components/LeftContents';
+import RightContents from './components/RightContents';
+import StickySummary from './components/rightSide/StickySummary';
+
+const GoodsDetail = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    fetch('/data/goodsData.json')
+      .then(res => res.json())
+      .then(res => setData(res));
+  }, []);
+
+  return (
+    <GoodsContents>
+      {data.map(data => (
+        <GoodsContainer key={data.id}>
```

Comment:
단순 최상위를 감싸는 용도로만 사용하신다면 Fragment를 사용하시면 됩니다.
```suggestion
        <React.Fragment key={data.id}>
```